Gruntfile.js

```diff
@@ -0,0 +1,43 @@
+module.exports = function(grunt) {
+    var BASE_URL = 'http://localhost:8000/mocha/';
+
+    /*
+     * To add a new app to test, add the app name to this test and create
+     * a test runner view at:
+     *
+     * corehq/apps/<app>/templates/<app>/spec/mocha.html
+     *
+     * You can view an example in the app_manager
+     */
+    var apps = [
+        'app_manager',
+    ];
+
+    var mochaConfig = {},
+        watchConfig = {};
+
+    apps.forEach(function(app) {
+        mochaConfig[app] = {
+            options: {
+                urls: [BASE_URL + app],
+                run: true
+            }
+        }
+        watchConfig[app] = {
+            files: [
+                'corehq/apps/' + app + '/static/' + app + '/js/**/*.js',
+                'corehq/apps/' + app + '/static/' + app + '/spec/**/*.js',
+            ],
+            tasks: ['mocha:' + app]
+        }
+    });
+
+    grunt.initConfig({
+        mocha: mochaConfig,
+        watch: watchConfig,
```

Comment:
similar to our sniffer, this will run on file changes. and it's fast!

<img width="650" alt="screen shot 2015-10-19 at 12 28 19 am" src="https://cloud.githubusercontent.com/assets/918514/10569713/5339931c-75f8-11e5-8e1b-cb5c8aa2eada.png">